README.md

```diff
@@ -153,27 +157,34 @@ In order to render license key form just add the following to your settings page
 > ⚠️ This will render license key fields for all of your registered plugins/services in the same Uplink/Container instance.
 
 ```php
-use StellarWP\Uplink\Config;
+use StellarWP\Uplink as UplinkNamespace;
 
-$fields = Config::get_container()->get( License_Field::class );
+$form = UplinkNamespace\get_form();
+$plugins = UplinkNamespace\get_plugins();
 
-// Do one of the following:
-$fields->render();               // Render the fields, titles, and submit button.
-$fields->render( false );        // Render the fields without the titles.
-$fields->render( false, false ); // Render the fields without the titles or submit buttons.
+foreach ( $plugins as $plugin ) {
+	$field = UplinkNamespace\get_field( $plugin->get_slug() );
+	// Tha name property of the input field.
+	$field->set_field_name( 'field-' . $slug );
+	$form->add_field( $field );
+}
+
+$form->render();
+// or echo $form->get_render_html();
 ```
 
 To render a single product's license key, use the following:
 
 ```php
-use StellarWP\Uplink\Config;
+use StellarWP\Uplink as UplinkNamespace;
+
+$resource = UplinkNamespace\get_resource( 'my-test-plugin' );
```

Comment:
Don't need this at all
```suggestion
```

Reply:
I do need this - im using it in line 182